Issue_Working_Guide.md

```diff
@@ -0,0 +1,23 @@
+# Guidelines for working on an Issue 
+
+**Note**: This guide will act as a set of rules, that are required to be followed by anyone who is going to work on an issue in this project.
+
+# Key Points
+1. Assignee
+2. Label
+3. Description
+4. Elementary
+
+## Assignee
+Before starting to work on an issue, make sure to add yourself as assignee of that issue.
+
+## Label
+While working on an issue, add appropriate labels on that issue. 
+Ex: *If you are working on issue branch add the label in Progress.*
+    *If the PR of issue is merged than add the label Done*
```

Comment:
Correct the spelling of _than_ to _then_
Also add the point to tag label **closed** when PR is closed rather than merged.

Reply:
okk